app/controllers/api/v1/recordings_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class RecordingsController < ApplicationController
+      # GET /api/v1/recordings.json
+      # Returns: { data: Array[serializable objects(recordings)] , errors: Array[String] }
+      # Does: Returns all the Recordings from all the current user's rooms
+      def index
+        recordings = current_user.recordings
+
+        render json: { data: recordings, status: :ok, include: :formats }
```

Comment:
This is still using `render json` - you should be using the `render_json` method instead